### DISCUSSION TITLE
personalized menu

### DISCUSSION THREAD

**Discussion by newtec-foggia** [oneduo/nova-file-manager#16] [Q&A] [answered]
<<Now that the tool is registered, if you go back to your Nova dashboard, you should see a new navigation entry labeled " File Manager".>>

I have a personalized menu, how do I proceed?
thank you

![menu](https://user-images.githubusercontent.com/83398776/184408863-f3c01d1b-b995-4dc2-9fe4-376651bfc623.JPG)

**Comment by mikaelpopowicz** [comment, marked as answer]
Hi @newtec-foggia

You may add a section like this

```php
MenuSection::make('File Manager')
    ->path('/nova-file-manager')
    ->icon('server'),
```

You also may add an item into a section like this
```php
MenuSection::make('Your section', [
    MenuItem::make('File Manager')
        ->path('/nova-file-manager'),
    ]),
```

**Reply by newtec-foggia** [reply]
Hi @mikaelpopowicz 
![ERRORE](https://user-images.githubusercontent.com/83398776/184425213-6ff1e363-c368-4361-b42e-4bcf5f8c6a2e.JPG)

**Reply by mikaelpopowicz** [reply]
Can you upgrade the package to 0.2.2 ?

**Reply by newtec-foggia** [reply]
upgraded to 0.2.2.. perfect thanks

**Comment by newtec-foggia** [comment]
Hi @mikaelpopowicz 
an error, but the upload to the folder I checked is ok..
i can't upgrade to laravel 9 because i have other components
![ERRORE](https://user-images.githubusercontent.com/83398776/184428674-7fbd0d31-bea7-48d5-a9b4-c0d40fc39f38.JPG)

**Reply by mikaelpopowicz** [reply]
Hi @newtec-foggia 

Please upgrade the package to 0.2.3.

**Reply by newtec-foggia** [reply]
perfect , thanks
